config/settings.yml

```diff
@@ -97,6 +97,7 @@
     :purge_window_size: 10000
 :ems:
   :ems_redhat:
+    :resolve_ip_addresses: true
```

Comment:
@adahms the change isn't visible in the UI. We are adding a configuration setting, but it will only be visible in the UI in the text editor of the configuration file: Configuration -> Advanced. I think we doesn't qualify as visibility in the UI.

Note that I wrote a Doc Text in the bug:

https://bugzilla.redhat.com/1417757

Maybe you can take a look at that, see if it is appropriate,  and tell us where that kind of information fits better. A release note? The "Managing Providers" guide?